platform/default/mbgl/map/map_snapshotter.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <mbgl/util/image.hpp>
+#include <mbgl/util/thread.hpp>
+#include <mbgl/util/optional.hpp>
+
+#include <exception>
+#include <string>
+#include <functional>
+
+namespace mbgl {
+
+template<class> class ActorRef;
+struct CameraOptions;
+class FileSource;
+class Size;
+class LatLngBounds;
+
+class SnapshotCallback {
```

Comment:
I mean keep `ActorRef<SnapshotCallback>` as it is, but have `SnapshotCallback` just be a `std::function` typedef. I believe that's possible with your change in 68f470fdda4e31d7704fba3e41bb2f899db39541.